start.sh

```diff
@@ -4,6 +4,7 @@
 set -ae
 
 function prepare_vector_sink() {
+	VECTOR_COMPRESSION_ENABLED=${VECTOR_COMPRESSION_ENABLED:-false}
```

Comment:
Why not default this to true? Particularly if it's for devices out in the field (rather than a server) it'll make a huge difference

Reply:
Since the Vector V2 endpoint does not need to be set to receive compressed requests, it does make sense for this to be default to `true`.  I'll add another commit.